src/v0/destinations/braze/transform.js

```diff
@@ -308,6 +309,37 @@ function processTrackEvent(messageType, message, destination, mappingJson) {
   payload = addMandatoryEventProperties(payload, message);
   payload.properties = properties;
 
+  try {
+    // add,update,remove
+    if (destination.Config.enableNestedArrayOperations) {
+      Object.keys(properties)
+        .filter(k => Array.isArray(properties[`${k}`]))
```

Comment:
```suggestion
        .filter(key => Array.isArray(properties[`${key}`]))
```

Reply:
using 'key' later , I think its fine to use 'k' for this filter block

Reply:
You can use key at both places